### PR TITLE
Fixes: Countdown Calculator Improvements

### PR DESCRIPTION
**Pull request by RayDeeUx**
Changes:
- Fix that one edge case where countdown calculations appear in the wrong places
> Before:
> image by jani
> After:
> image by ery

- Bring display names to the "Special" column of the Calendar Overlay
> image
> image
> image
...etc, etc.

- Rewrote the way `useFormatter` was processed so that years are displayed on countdowns lasting over a year
> ~~[i was *going* to get an image but alpha hypixel decided to shut down so enjoy this quote block instead]~~
> image

- Changed when countdown calculations are displayed in the countdown overlay
> Before:
> image
> After:
> image
> image

### CODE REVIEW

**Comment by RayDeeUx** [comment]
irrelevant:
- moved kotlin files away from java directory to kotlin directory
- added punctuation to star cult description

**Comment by RayDeeUx** [comment]
(and no, i haven't added an inline option here yet. part of this is because the calendaroverlay.java code is no longer spaghetti code, but rather a full-blown five-course dinner)

**Comment by RayDeeUx** [comment]
(that isn't to say i wouldn't eat said metaphorical five-course dinner *at all*, but rather just not in one sitting.)

(or someone else can finish the meal instead, who knows.)

**Comment by RayDeeUx** [comment]
"omg ery, what about this weird plus sign for the year displayed in the countdown calculations?!"
image
i *did* use an extreme example here; i doubt anyone's going to have that long of a booster cookie duration on the live hypixel network. at first i thought it was my mistake for using `YYYY` instead of `yyyy` on the DateTimeFormatter class, but apparently not.

tl;dr: extreme edge case, easily disregarded

**Review by nea89o — APPROVED** [review]

**Review by NopoTheGamer — APPROVED** [review]
